pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -170,8 +174,8 @@ func TestShowChangefeedJobsRedacted(t *testing.T) {
 			expectedSinkURI = strings.Replace(expectedSinkURI, certSecret, "redacted", 1)
 			expectedDescription := strings.Replace(createStmt, apiSecret, "redacted", 1)
 			expectedDescription = strings.Replace(expectedDescription, certSecret, "redacted", 1)
```

Comment:
expectedDescription is written twice here.

Reply:
I will leave it as it is for now : )

Reply:
I went with the second option here. Lmk if you are against it.